homeassistant/components/denonavr/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "denonavr",
   "name": "Denon AVR Network Receivers",
   "documentation": "https://www.home-assistant.io/integrations/denonavr",
-  "requirements": ["denonavr==0.8.0"],
+  "requirements": ["denonavr==0.8.1"],
   "dependencies": [],
-  "codeowners": []
+  "codeowners": ["@scarface-4711", "@starkillerOG"]
```

Comment:
You added @starkillerOG here as code owner. Is he aware?

Reply:
I was not, but now I am :)
Thank you for tagging me!
I am fine with beeing codeowner of denonavr since I regularly push commits to it anyway.